docs/GettingStarted.md

```diff
@@ -64,7 +64,8 @@ of the Datadog tracer, you can override the following defaults:
       tracer: Datadog.tracer,
       debug: false,
       trace_agent_hostname: 'localhost',
-      trace_agent_port: 8126
+      trace_agent_port: 8126,
+      env: Rails.env
```

Comment:
OK, I think I see your point. I, too, feel at first incomfortable with having two ways to do the same thing. But, if you think about it: it's going to be stored as a tag after all... What I'd say is that this is a good candidate for a `.merge` of Ruby objects containing these infos. I view it as something similar as the config of our agent. You can configure some parts of it with `ENV`, some with the config, at some point those infos are merged. It's still nice to have this `env` parameter sticking out as, really, it's different from other tags: it has a special meaning in our backend, and shows up as a special dropdown in the interface. So it matters, it's not like other tags which we do nothing with. At some point I'm wondering why it's not a first class member here: https://github.com/DataDog/datadog-trace-agent/blob/master/model/span.go#L17 The only thing is we need to be clear about this merging mecanism in the docs. But if we drop one of the two methods, something is broken I think.